sdmetrics/reports/single_table/_properties/synthesis.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+import pandas as pd
+import plotly.express as px
+
+from sdmetrics.reports.single_table._properties import BaseSingleTableProperty
+from sdmetrics.single_table import NewRowSynthesis
+
+
+class Synthesis(BaseSingleTableProperty):
+    """Synthesis property class for single table.
+
+    This property assesses the novelty of the syntetic data over the real data.
+    The ``NewRowSynthesis`` metric is computed over the real and synthetic table to
+    score the proportion of new rows in the synthetic data.
+    """
+
+    metric = NewRowSynthesis
+
+    def _generate_details(self, real_data, synthetic_data, metadata, progress_bar=None):
+        """Generate the _details dataframe for the synthesis property.
+
+        Args:
+            real_data (pandas.DataFrame):
+                The real data
```

Comment:
We usually end args with dots. Same for args below.

Reply:
Yes, done in [b00d7ca](https://github.com/sdv-dev/SDMetrics/pull/398/commits/b00d7cafd54bf24f3b76c78763bddbd65d5b9408)